examples/source/1.components/amp-story-shopping.html

```diff
@@ -0,0 +1,267 @@
+    <meta name="description" content="This is an Web Story with a shopping experience.">
+    <link rel="preload" as="script" href="https://cdn.ampproject.org/v0.js">
+    <link rel="preload" as="script" href="https://cdn.ampproject.org/v0/amp-story-1.0.js">
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <!-- To begin, import `amp-story-shopping` along with your other stories imports. -->
+    <script async custom-element="amp-story-shopping" src="https://cdn.ampproject.org/v0/amp-story-shopping-0.1.js"></script>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <link rel="canonical" href=".">
+    <title>amp-story-shopping</title>
+    <style amp-custom>
+      [data-product-id="lamp"] {
+        top: 20%;
+        left: 21%;
+      }
+      [data-product-id="chair"] {
+        top: 53%;
+        left: 30%;
+      }
+      [data-product-id="flowers"] {
+        top: 46%;
+        left: 83%;
+      }
+    </style>
+  </head>
+  <body>
+    <amp-story standalone
+      publisher="Stories Format"
+      title="amp-story-shopping"
+      poster-portrait-src="https://source.unsplash.com/_HqHX3LBN18/1000x2000"
+      publisher-logo-src="https://amp.dev/static/img/icons/icon-512x512.png">
+      <!-- 
+        ## Basic usage
+
+        `amp-story-shopping` is defined in the document using its two custom elements, [`amp-story-shopping-attachment`](/documentation/components/amp-story-shopping#amp-story-shopping-attachment/?format=stories) and [`amp-story-shopping-tag`](/documentation/components/amp-story-shopping#amp-story-shopping-tag/?format=stories).
+
+        Specify a shopping experience by defining one [`amp-story-shopping-attachment`](/documentation/components/amp-story-shopping#amp-story-shopping-attachment/?format=stories) as the last element on the page.
+        The `amp-story-shopping-attachment` component accepts 1 descendant, a script element that with product JSON that configures the experience.
+
+        Product JSON must be configured inline with the script child by default. It can also be loaded remotely with a `src` tag on the `amp-story-shopping-attachment` element.
```

Comment:
This section is confusing to me - it must be defined inlined but also loaded remotely?